jina/proto/jina.proto

```diff
@@ -266,6 +266,7 @@ message RequestProto {
         ControlRequestProto control = 5; // a control request
         DeleteRequestProto delete = 9; // a delete request
         UpdateRequestProto update = 10; // an update request
+        DumpRequestProto dump = 11; // an dump request
```

Comment:
What is the view on how this may evolve? Any new Indexer that is added to the hub will need a new Dump Format?

Reply:
No, the dump format is universal right now. Check the enum types, `DumpFormats`. So right now you can dump and import into any. 

Yes, we can add specific formats where you can optimize the format for importing etc.